org/wildfly/undertow/http-invoker/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.undertow.http-invoker
+
+DESCRIPTION: Entry point for remote http invoker, supporting ejb over http
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: false
```

Comment:
for now there are no other usages, but I think there might be outside usages coming soon.
Changed it to true, for now.